manifests/init.pp

```diff
@@ -108,4 +108,6 @@
   $datastore_keys_dir       = $::st2::params::datstore_keys_dir,
   $datastore_key_path       = "${::st2::params::datstore_keys_dir}/datastore_key.json",
   $nginx_manage_repo        = true,
```

Comment:
Please add documentation to this class about the new parameters

Reply:
Added comments about the new chatops variables.